aea/aea.py

```diff
@@ -89,6 +90,8 @@ def __init__(
         connection_exception_policy: ExceptionPolicyEnum = ExceptionPolicyEnum.propagate,
         loop_mode: Optional[str] = None,
         runtime_mode: Optional[str] = None,
+        default_ledger: Optional[str] = None,
+        ledger_id_to_currency_denom: Optional[Dict[str, str]] = None,
```

Comment:
Not sure about the name. Maybe just `currency_denominations`?

Reply:
`amount` instead of `denomination`?

Reply:
yes `currency_denominations` is preferable since it is shorter, the fact that it's a map comes from the type hint

Reply:
but both works at the end